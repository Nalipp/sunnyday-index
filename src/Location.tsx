import { ILocation } from './interfaces';

interface LocationProps {
  location: ILocation
}


function Location({ location }: LocationProps) {
  return (
    <li>
      <p>{location.cityName}</p>
      <p>{location.lng}</p>
    </li>
  );
}

export default Location;
