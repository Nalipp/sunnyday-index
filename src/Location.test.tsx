import { render } from '@testing-library/react';
import Location from './Location';
import locations from './defaultData';

const location = locations[0];

test('renders Location component', () => {
  render(<Location location={location} />);
});

test('Location component matches snapshot', () => {
  const { container } = render(
      <Location location={location} />
    );

  expect(container).toMatchSnapshot();
})
