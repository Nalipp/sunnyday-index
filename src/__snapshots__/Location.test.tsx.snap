// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Location component matches snapshot 1`] = `
<div>
  <li>
    <p>
      Cancun
    </p>
    <p>
      -86.840317
    </p>
  </li>
</div>
`;
